integration_tests/test_basic.py

```diff
@@ -30,7 +30,6 @@
 def test_basic(cluster):
     w3 = cluster.w3
     assert w3.eth.chain_id == 777
-    assert w3.eth.get_balance(ADDRS["community"]) == 10000000000000000000000
```

Comment:
it only works if this is the very first test case to run, but it's not guaranteed.